src/Observable.ts

```diff
@@ -255,3 +256,9 @@ export class Observable<T> implements CoreOperators<T>  {
   zip: <R>(...observables: Array<Observable<any> | ((...values: Array<any>) => R)>) => Observable<R>;
   zipAll: <R>(project?: (...values: Array<any>) => R) => Observable<R>;
 }
+
+//let import occurs after Observable<T> is exported
+import {ArrayObservable} from './observable/fromArray';
```

Comment:
This resolves https://github.com/ReactiveX/RxJS/issues/894, but import occurs after `Observable` is actually declared to let `ArrayObservable` able to reference `Observable`.